xpt_neo_training.py

```diff
@@ -338,10 +345,11 @@ def reset_(self):
                     f"{phase_detector()}-"
                     f"{config['experiment']['type']}-"
                     f"{config['experiment']['name']}-"
-                    f"steps={CURRENT_STEP}",
+                    f"steps={REAL_STEP}",
                 )
                 model.save_pretrained(save_dir)
-                engine.save_checkpoint(os.path.join(save_dir, "deepspeed"))
+                # engine.save_checkpoint(os.path.join(save_dir, "deepspeed"))
                 CURRENT_STEP = total_num_steps
 
-        CURRENT_STEP += 1
\ No newline at end of file
+        CURRENT_STEP += 1
```

Comment:
What's different `CURRENT_STEP` and `REAL_STEP`?

Reply:
Current_step is phase 1, 2 step, REAL_STEP is the real step for EarlyStopping counting